README.md

```diff
@@ -1,13 +1,15 @@
 # SSE R-plugin
 
-Server Side Extension (SSE) is a general technology for extending the Qlik built in expression library, both for load-script and for chart expressions with functionality from external calculation engines. The main purpose is to use SSE in Qlik visualization measures and to some extent also for calculated dimensions.
+Server Side Extension (SSE) is a general technology for extending the Qlik built in expression library, both for load-script and for chart expressions with functionality from external calculation engines. The main purpose is to use SSE in Qlik visualization measures and to some extent also for calculated dimensions.  
 
-R is not a supported language in gRPC by default. The purpose of this repository is therefore to provide a possible solution using a C# SSE plugin which in turn access Rserve to be able to run R scripts. All documentation and guides regarding SSE in general, referred to as server-side-extension, are valid and useful for this plugin as well.
+SSE is part of the Advanced Analytics Integration (AAI) concept at Qlik.  
+
+R is not a supported language in gRPC by default. The purpose of this repository is therefore to provide a possible solution using a C# SSE plugin which in turn access Rserve to be able to run R scripts. All documentation and guides regarding SSE in general, referred to as server-side-extension, are valid and useful for this plugin as well.  
 
 ## Status
 **Current Plugin Version and State:** v1.0.0  
 **Matching Qlik Sense Version:** Qlik Sense 2017 release (or later). Both desktop and enterprise.  
-**Disclaimer:** This plugin is not meant to be used in production, therefore **no support is provided**. Use it at your own risk.  
+**Disclaimer:** Use it at your own risk. See LICENSE.txt file.  
```

Comment:
Perhaps make [LICENSE.txt](LICENSE.txt) into a link? Also, shouldn't it be "See [LICENSE.txt](LICENSE.txt)" or "See the [LICENSE.txt](LICENSE.txt) file" ?